packages/shared/src/components/modals/NewSourceModal.tsx

```diff
@@ -236,8 +236,24 @@ export default function NewSourceModal(props: ModalProps): ReactElement {
     return <PushNotificationModal {...modalProps} />;
   }
 
+  const linkHasFeed = !!feeds?.length;
+
   return (
-    <Modal {...modalProps}>
+    <Modal
+      {...modalProps}
+      formProps={{
```

Comment:
Too late now, but djeez this gives me headaches 😂
Really becomes messy for some reason all the logic and conditional stuff.